sahi/model.py

```diff
@@ -1063,3 +1063,106 @@ def _create_object_prediction_list_from_original_predictions(
             object_prediction_list_per_image.append(object_prediction_list)
 
         self._object_prediction_list_per_image = object_prediction_list_per_image
+
+
+@check_requirements(["tensorflow", "tensorflow_hub"])
+class TensorflowHubDetectionModel(DetectionModel):
+    def load_model(self):
+        try:
```

Comment:
1068 satirda zaten check ediyoruz, classin icinde tekrar check etmeye gerek yok.

```suggestion

```

Reply:
Check işlemlerini class içinden sildiğimizi yeni gördüm. Tensorflow için bu işlemleri silmekten neden vazgeçtik?

Reply:
silmekten vazgecmedik, silinmesini istiyoruz. sahi'ye ekledigimiz yeni ozellik sayesinde class ve fonksiyonlarin uzerine bu decorator'u eklemek yeterli oluyor: `@check_requirements(["tensorflow", "tensorflow_hub"])`

Reply:
Aşağıda vazgeçtim yazısını görünce iptal ettiğimizi düşünmüştüm. Düzeltiyorum hemen.